gui.go

```diff
@@ -926,6 +928,11 @@ func (g *Gui) execKeybindings(v *View, ev *gocuiEvent) (matched bool, err error)
 	return false, nil
 }
 
+// Set the style of the cursor
+func (g *Gui) SetCursorStyle(style CursorStyle) {
```

Comment:
```suggestion
// SetCursorStyle sets the style of the cursor
func (g *Gui) SetCursorStyle(style CursorStyle) {
```

Reply:
I opened the PR for the discussion but sure ;)